recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/wayland-ivi-extension_2.0.0.bbappend

```diff
@@ -1,2 +1,2 @@
 SRC_URI = "git://github.com/GENIVI/${BPN}.git;protocol=http"
-SRCREV = "2.2.0"
+SRCREV = "2.2.0_weston_5_compartible"
```

Comment:
just curious why this specific branch?

Reply:
So, basically we have 3 options:

1. Leave `2.2.0` - then project will not build
2. Use this tag - and we are good to go
3. Use master branch

Taking into account that receipt is named blah-blah-blah_2.0.0, neither option is correct.

Personally, I'm sticking to option 2 + rename receipt to reflect real version.

Reply:
Done

Reply:
Appropriate comment could be an option here